sdcm/sct_config.py

```diff
@@ -1222,6 +1222,10 @@ class SCTConfiguration(dict):
         dict(name="availability_zone", env="SCT_AVAILABILITY_ZONE",
              type=str,
              help="Availability zone to use. Same for multi-region scenario."),
+        dict(name="auto_availability_zone", env="AUTO_AVAILABILITY_ZONE",
```

Comment:
the environment variable should be prefixed with `SCT_`

also it AWS only feature, so it's config option should be prefixed with `aws`

Reply:
Also add a default `False` in `test_default.yaml` for this configuration

Reply:
fixed